packages/tic-tac-toe-bot/scripts/postinstall.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -z "${IS_HEROKU_ENV}" ]; then
+    echo "📟  Detected Heroku. Transpiling Typescript into JS for Node.JS to run on server."
+    tsc -p tsconfig.heroku.json &> /dev/null || :
+fi
```

Comment:
Hey by the way, coming soon Heroku is going to auto-run `yarn build` so we may be able to use that